shell.nix

```diff
@@ -0,0 +1,17 @@
+{ pkgs ? import <nixpkgs> { } }:
+pkgs.mkShell {
+  buildInputs = with pkgs; [
+    (python3.withPackages (ps: with ps; [ flask ]))
+    imagemagick
+    pdfsandwich
+    poppler_utils
+    which
+    ghostscript
+    netpbm
+
+    (pkgs.writers.writeBashBin "run" ''
```

Comment:
How about calling the run script run-dms?

Reply:
This is only for development, the commands usually "run" "build" "test" "clean"